File: money-service/src/main/java/com/hmpay/money/application/service/RechargeMoney.java
package com.hmpay.money.application.service;


import com.hmpay.common.UseCase;
import com.hmpay.money.application.port.in.RechargeMoneyCommand;
import com.hmpay.money.application.port.in.RechargeMoneyUseCase;
import com.hmpay.money.application.port.out.RechargeMoneyPort;
import jakarta.transaction.Transactional;
import lombok.RequiredArgsConstructor;



@RequiredArgsConstructor
@UseCase
@Transactional
public class RechargeMoney implements RechargeMoneyUseCase {

	private final RechargeMoneyPort rport;

	@Override
	public void rechargeMoney(RechargeMoneyCommand command) {

	}
}

File: payment-service/src/main/kotlin/com/hmpay/payment/application/service/RechargeMoney.kt
package com.hmpay.payment.application.service

import com.hmpay.common.UseCase
import com.hmpay.payment.application.port.`in`.RechargeMoneyCommand
import com.hmpay.payment.application.port.`in`.RechargeMoneyUseCase
import jakarta.transaction.Transactional

@UseCase
@Transactional
class RechargeMoney : RechargeMoneyUseCase {
    override fun rechargeMoney(command: RechargeMoneyCommand) {
        // TODO: implement recharge logic
    }
}
